Security.Platform.Android/AgsSec.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using AeroGear.Mobile.Core.Utils;
+
+namespace AeroGear.Mobile.Security
+{
+    public class AgsSec
```

Comment:
not sure I like this name. The name space already contains "AeroGear", can we just name it "SecurityService"?